### PR TITLE
Fix #9434: Added Insert One Measure icon

### LINKED ISSUE #9434: [Inspector] Vertical, Horizontal and Text frames & 'Insert measure' do not have icons in the Palettes menu
**Describe the bug**
If you observe the Palettes menu in 'layout' in MS4, you'll see that the **Vertical, Horizontal** and **Text** frame options do not display correctly
image

In MS3, they look like this:
image

If new icons are needed, these can be provided. Let us know!

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #9434 

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/framework/ui/view/iconcodes.h [hunk 1]
```diff
@@ -302,7 +302,7 @@ class IconCode
 
         TOOLBAR_GRIP = 0xF3A2,
 
-        PAGE = 0xF3A5,
+        PAGE = 0xF3AE,    //f3a5
```
Comment (cbjeukendrup) [inline src/framework/ui/view/iconcodes.h:305]: What's this?

Comment (Jojo-Schmitz) [reply src/framework/ui/view/iconcodes.h:305]: Same as Camera???

Comment (HemantAntony) [reply src/framework/ui/view/iconcodes.h:305]: Sorry, I was testing something else. I forgot to change it back

**Comment by bkunda** [comment]
Tested on Win and Mac – new icon appears where it should be.

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**